src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class FigureSupplier extends ColorSupplier {
```

Comment:
```suggestion
public class FigureSupplier {
    private ColorSupplier colorSupplier = new ColorSupplier();
```
The same here they have different purpose. You can use one inside of another but it won't be correct to extend it.